tools/osx/crosstool/BUILD.tpl

```diff
@@ -50,6 +50,7 @@ cc_toolchain_suite(
             ":builtin_include_directory_paths",
             ":cc_wrapper",
             ":libtool",
+            ":libtool_check_unique",
```

Comment:
+1 on this one, seems like this label references the binary in the toolchain setup